scraper/args.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+from argparse import Namespace
+from logging import _nameToLevel
+import enum
```

Comment:
Are we using anything else from enum other than Enum? If not let's just do 
from enum import Enum